getting-started-github/README.md

```diff
@@ -0,0 +1,37 @@
+
+You're done! You just made a file, committed it, and pushed it to the remote repository. If you refresh the browser page that holds your online repo, you should see the file you made along with your unique commit message. 
+
+## How does my team work together?
+This is a little bit of an advanced topic, but its integral if you are working on a team. The obvious flaw with the method I described earlier is if multiple people attempt to push conflicting changes to the same repository at the same time. **Branches** solve this problem in a pretty elegant way. To understand them way better than I could ever explain, read [this](https://codesource.io/understanding-branching-in-git/). 
+
+For my teams, we have one repo and everyone is added on as a [collaborator](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository). Next, we have everyone make their own branch and we keep the **master** branch seperate. This makes sure no one pushes untested code directly to our current product. Everyone pushed to their own branch, and then makes a **pull request** to master. To learn how to do this, follow [this](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request). This way, everyone can review the proposed changes before **merging** (combining) the changes with the main master branch. 
+
+Remember to **pull** changes from the remote branch to your local copy whenever your teammate makes a change. This ensures that everyone has the most up to date version of the code, and no one is working on an already solved bug. 
+
+## Where can I learn more?
+This is only the tip of the iceberg when it comes to Git and Github. Once you gain experience and confidence, you'll find yourself using git functions like rebase, fetch, diff, tag and config. To learn more about these, here are a couple tutorials I'd recommend. 
+
+1. [Good slideshow for complete beginners] (https://www.slideshare.net/HubSpot/git-101-git-and-github-for-beginners) and the [corresponding blog post](https://product.hubspot.com/blog/git-and-github-tutorial-for-beginners)
+2. [Pretty comprehensive tutorials on git, but no github specific info](https://www.atlassian.com/git/tutorials). If you're using github, then these mostly serve to help you learn the theory behind git. 
+3. Cool interactive [tool](https://learngitbranching.js.org) hosted on....Github! HIGHLY RECOMMENDED for people confused by my rambling tutorial. 
+
+Before hopping too deep into any of these though, I'd recommend just going up and asking a mentor/company rep for help. Odds are that they use Github (or another git hosting service) at work, and can probably solve your problem within minutes. This will give you more time to focus on building.
```

Comment:
```suggestion
Before hopping too deep into any of these though, I'd recommend just going up and asking a mentor/company rep for help. Odds are that they use Github (or another git hosting service) at work, and can probably solve your problem within minutes. This will give you more time to focus on building. Git tends to be one of those things that has a high learning curve so don't be too distraught if it's confusing. It'll come much more naturally as you use it more!
```